sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_appconfiguration_client_async.py

```diff
@@ -126,16 +129,16 @@ def list_configuration_settings(
     ) -> AsyncItemPaged[ConfigurationSetting]:
 
         """List the configuration settings stored in the configuration service, optionally filtered by
-        label and accept_datetime
+        key, label and accept_datetime.
 
         :param key_filter: filter results based on their keys. '*' can be
-         used as wildcard in the beginning or end of the filter
+            used as wildcard in the beginning or end of the filter
         :type key_filter: str
         :param label_filter: filter results based on their label. '*' can be
-         used as wildcard in the beginning or end of the filter
+            used as wildcard in the beginning or end of the filter
         :type label_filter: str
         :keyword datetime accept_datetime: filter out ConfigurationSetting created after this datetime
-        :keyword List[str] fields: specify which fields to include in the results. Leave None to include all fields
+        :keyword list[str] fields: specify which fields to include in the results. Leave None to include all fields
```

Comment:
I think it should be typing.List here?

Reply:
It is in lowercase `list` which means the built-in type used in code instead of typing.List.